javaagent-bootstrap/src/main/java/io/opentelemetry/javaagent/bootstrap/AgentInitializer.java

```diff
@@ -100,31 +98,8 @@ public static ClassLoader getExtensionsClassLoader() {
    *     classloader
    * @return Agent Classloader
    */
-  private static ClassLoader createAgentClassLoader(String innerJarFilename, File javaagentFile)
-      throws Exception {
-    ClassLoader agentParent;
-    if (isJavaBefore9()) {
-      agentParent = null; // bootstrap
-    } else {
-      // platform classloader is parent of system in java 9+
-      agentParent = getPlatformClassLoader();
```

Comment:
Any idea why we were doing this before?

Reply:
this was inherited, my best guess is for access to `java.sql` package for some reason(?)